deliverables/README.md

```diff
@@ -8,61 +10,122 @@ To answer questions, please just put the answer on the next line after the quest
+
+
+**What is a div?**   <br/>is the generic container for flow content, which does not inherently represent anything. It can be used to group elements for styling purposes. 
+
+CSS
+**What are selectors?**   <br/>The rule set that actually selects the content you want to style.
+
+
+**What are properties?**  <br/>A property is like a tag in HTML. These are the items that define how the styles should look on the Web page or elsewhere.
+
+
+**What are values?**  <br/>A value is given to the property following a colon and semi-colons separate the properties.
+
+
+**How do CSS styles for a particular element get inherited?** <br/>ie. how does an element get its "default" styles?  
+
+
+**What are two CSS attributes you can change to push an element around on the page?**  Position; relitive, Position; absolute. 
+
+
+**What are the three different ways to include CSS in your project or use CSS to style a particular element?**    Inline, Embedded, and External Style Sheets.
+
+
 
----
 ### Section 3: Git and Github  
-What is Git?  
-What is SCM?  
-What is a VCS?  
-Why is Git useful for a developer?  
-Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
-How do you create it on Github?  
-How do you commit changes?  
-What is the difference between staging and committing changes?  
-What is the difference between committing your changes and pushing them to Github?  
-What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
-How can you look through your historical commits on the Github website?  
-What is a "Merge"?  
-What is a "Pull Request"?  
-What is "Forking" a repo?  
-What is "Cloning" a repo?  
+**What is Git?**  <br/>It is an open source program for tracking changes in text files. It was written by the author of the Linux operating system, and is the core technology that GitHub, the social and user interface, is built on top of.
+
+
+**What is SCM?** <br/>Supply chain management software, is the software tools or modules used in executing supply chain transactions, managing supplier relationships and controlling associated business processes. 
+
+
+**What is a VCS?**  <br/>Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later.
+
+
+**Why is Git useful for a developer?**  <br/>It gives him a place where he can push things up to his or her live site. 
+
+
+**Why is Git useful for a team of developers?**  <br/>So everyone can work on the same project simultaneously. 
+
+
+**How do you create a new Git repository for a project locally?**  
+Open git-hub and hit the + button in the top left and create a anew repo by chosing where you want the repo to be on your computer. 
```

Comment:
Are you referring to a Github desktop client?  If so this is correct.  What we were looking for here is how you would do it from a regular command line.  Answer is `git init`